Code/GraphMol/AddHs.cpp

```diff
@@ -516,8 +516,21 @@ void setTerminalAtomCoords(ROMol &mol, unsigned int idx,
   }
 }
 
-void addHs(RWMol &mol, bool explicitOnly, bool addCoords,
-           const UINT_VECT *onlyOnAtoms, bool addResidueInfo) {
+namespace {
+bool isQueryAtom(const RWMol &mol, const Atom &atom) {
```

Comment:
hm, I'm not quite happy about this function name (as we also return `true` for regular atoms with query bonds). But I can't think of anything better...